editor.planx.uk/src/@planx/components/PlanningConstraints/Public.tsx

```diff
@@ -71,9 +77,10 @@ function Component(props: Props) {
 
   // Check if this team should query Planning Data (or continue to use custom GIS) and set URL params accordingly
   //   In future, Planning Data will theoretically support any UK address and this db setting won't be necessary, but data collection still limited to select councils!
-  const hasPlanningData = useStore(
-    (state) => state.teamIntegrations?.hasPlanningData,
-  );
+  const hasPlanningData = true;
+  // useStore(
+  //   (state) => state.teamIntegrations?.hasPlanningData,
+  // );
```

Comment:
:warning: this is a temporary hack so that we can fetch planning constraints in the editor sidebar & view automations in the graph view side-by-side for easier testing :warning: 

Currently, `state.teamIntegrations` is undefined in the editor view - adding an `updateTeamIntegrations` method to the `TeamStore` & planx-core looks like the best route to permanently fix this.

Reply:
Thanks for `initTeamStore()` pointer - that's a much simpler solution than I originally anticipated ! 

Fixed here https://github.com/theopensystemslab/planx-new/pull/3458 (so we can get to `main` sooner/independently and I'll rebase this one)